src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Networkplugin syncer"
+---
+
+The **_networkplugin-syncer_** interfaces to the K8s network plugin subsystem
```

Comment:
@mangelajo thank you for working on this.

Can you please include some notes on the purpose of networkplugin-syncer when we already have route-agent drivers?
This will help people to understand the role of networkplugin-syncer and route-agent when both are installed from a high-level.